qiskit_experiments/library/characterization/analysis/cr_hamiltonian_analysis.py

```diff
@@ -209,6 +173,14 @@ def _default_options(cls):
             legend_loc="lower right",
             fit_report_rpos=(0.28, -0.10),
             ylim=(-1, 1),
+            plot_options={
+                "x|c=0": {"color": "blue", "symbol": "o", "canvas": 0},
```

Comment:
Is this format documented somewhere?

Reply:
This is stated in the option description. I also added description of this option in release note.